tools/bcp47.py

```diff
@@ -32,8 +35,6 @@
     "canadien" : "fr-CA",
     "catalan" : "ca",
     "chinese" : "zh",
-    "chinese" : "zh-CN",
-    "chinese" : "zh-TW",
```

Comment:
Why these deletions?

Reply:
Python dictionaries can't have the same key map to multiple values. In the case of a conflict, the _last_ value defined is taken. For example,
```python
>>> {"chinese": "zh", "chinese": "zh-CN", "chinese": "zh-TW"}
{'chinese': 'zh-TW'}
```
[Babel](https://github.com/latex3/babel/blob/dcd551427599879303b7a047b6fd3aef0a77120e/locale/zh/babel-zh.ini#L19-L20) uses `chinese` to mean `zh`, so the other two have been removed.